example/cumulus-tf/ingest_and_publish_granule_with_orca_workflow.asl.json

```diff
@@ -290,6 +290,7 @@
             "cmr": "{$.meta.cmr}",
             "launchpad": "{$.meta.launchpad}",
             "etags": "{$.meta.file_etags}",
+            "skipMetadataValidation": true,
```

Comment:
This looks good but I am wondering, why not add this to other `*workflow.asl.json` files?

Reply:
I want to verify that task works when `skipMetadataValidation ` is not set

Reply:
Just happening to look through this, but I'm wondering why we're testing specific workflow configuration in an integration test at all -- for the purposes of integration this only is testing that the config object gets passed to the function, and I suppose in concert with the other task options, but this feels like an 'accidental' behavior in the overall ingest test rather than a specific, targeted, test. 

  Perhaps more importantly, if we -are- going to enable this in integration tests, shouldn't we default it to *not* hit CMR except for a test where it does to be kind re: their load issues?

Reply:
https://github.com/nasa/cumulus/pull/2897/commits/b7e90154a4230abc795db69450a79f78e7244ae4